lib/lwm2m-server.js

```diff
@@ -103,6 +104,29 @@ function validateTypes(config, callback) {
     callback(error);
 }
 
+/**
+ * If Lifetime Resource exists, then the registration SHOULD be removed by the Server if a new registration or update
+ * is not received within this lifetime.
+ */
+function checkLifetime() {
+    var CHECK_INTERVAL = 1000;
```

Comment:
This interval should be part of the configuration file, as it might be a parameter people would want to adapt to their needs.